### ISSUE TITLE
How can i add more more objects to the sketch ?

### ISSUE THREAD

**Issue by parkzinc** [noyoshi/smart-sketch#9]
Its possible add more like animals, etc ?

**Comment by noah-yoshida** [comment]
Yeah, you can. Just need to add label mapping that map the color the user uses, to the color the model is expecting. You can go to the NVIDIA repo and find the COCO stuff dataset greyscale colors that they associate with each class.
